src/software/backend/simulator_backend.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#include "shared/parameter/cpp_dynamic_parameters.h"
+#include "shared/proto/robot_log_msg.pb.h"
+#include "shared/proto/robot_status_msg.pb.h"
+#include "shared/proto/tbots_software_msgs.pb.h"
+#include "software/backend/backend.h"
+#include "software/backend/ssl_proto_client.h"
+#include "software/networking/threaded_proto_udp_listener.h"
+#include "software/networking/threaded_proto_udp_sender.h"
+#include "software/proto/defending_side_msg.pb.h"
+
+class SimulatorBackend : public Backend
+{
+   public:
+    SimulatorBackend(std::shared_ptr<const BackendConfig> config);
+
+
+   private:
+    void onValueReceived(TbotsProto::PrimitiveSet primitives) override;
+    void onValueReceived(World world) override;
+
+    /**
+     * Joins the specified multicast group on the vision_output, primitive_output
+     * and robot_status_input. Multicast Channel and Multicast Group are used
+     * interchangeably.
+     *
+     * NOTE: This will terminate the existing connection on the previous channel
+     * if it exists.
+     *
+     * @param channel The channel to join, index of MULTICAST_CHANNELS in
+     * shared/constants.h
```

Comment:
nit: change this comment

Reply:
changed and updated the one in wifi_backend.h